### ISSUE TITLE
Support live previews in Chrome/Chromium

### ISSUE THREAD

**Issue by tdulcet** [rugk/unicodify#52]
## Background
Live previews are not available in Chrome/Chromium because [`menus.onShown`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/onShown) and [`menus.refresh()`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/refresh) are not supported.

## Proposed solution
Implement this workaround: https://stackoverflow.com/a/13673942 (which was originally written by [a Mozilla employee](https://blog.mozilla.org/addons/author/robrobwu-nl/)).

## Additional context
This would be good to fix with #44, since the context menu code will need to be refactored anyway for that.

**Comment by rugk** [comment]
Good idea. An implementation note: We should use this as a fallback only for Chrome/ium, all other browsers (i.e. feature-detect whether `menus.onShown` is implemented/there) can continue to use the IMHO cleaner current implementation.
